Spark-IPython-64bit/ipython-pyspark.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+# https://github.com/felixcheung/vagrant-projects
+
+import os
+import sys
+
+#-----------------------
+# PySpark
+#
+
+master = 'local[*]'
+
+num_executors   = 12 #24
+executor_cores  = 2
+executor_memory = '1g' #10g
+
+pyspark_submit_args = os.getenv('PYSPARK_SUBMIT_ARGS', None)
+if not pyspark_submit_args:
+    pyspark_submit_args = '--num-executors %d --executor-cores %d --executor-memory %s' % (num_executors, executor_cores, executor_memory)
+pyspark_submit_args = '--master %s %s' % (master, pyspark_submit_args)
+
+if not os.getenv('PYSPARK_PYTHON', None):
+    os.environ['PYSPARK_PYTHON'] = sys.executable
+os.environ['PYSPARK_DRIVER_PYTHON']='ipython' # PySpark Driver (ie. IPython)
+
+ip = '*' # Warning: this is potentially insecure
+port = 1088
+
+os.environ['PYSPARK_DRIVER_PYTHON_OPTS'] = 'notebook --ip=%s --port=%s --no-browser --notebook-dir=/vagrant' % (ip, port)
+
```

Comment:
You are correct. Matplotlib inline doesn't work without that. But I thought it is simpler to just add `%matplotlib inline` to the beginning of a notebook to activate the inline mode.